content/en/profiler/enabling/java.md

```diff
@@ -29,7 +29,7 @@ As of dd-trace-java version 1.0.0, you have two options for the engine that gene
 
 Minimum JDK versions:
 
-- OpenJDK 8u352+, 11.0.17+, 17.0.5+
+- OpenJDK 8u352+, 11.0.17+, 17.0.5+ (including builds on top of it like Amazon Corretto, Azul Zulu, ...)
```

Comment:
```suggestion
- OpenJDK 8u352+, 11.0.17+, 17.0.5+ (including builds on top of it, including Amazon Corretto, Azul Zulu, and others)
```

Reply:
I just realized this is in the `documentation` repo, so I'll commit this suggestion. Let me know if you disagree with it and we can update it later.